packages/api-client/src/http/HttpClient.ts

```diff
@@ -245,6 +246,17 @@ export class HttpClient extends EventEmitter {
     return response.data;
   }
 
+  /**
+   * Will associate specified client id with session and return newly created access token.
+   *
+   * @param  {string} clientId - id of the client with which the new login session will be associated
+   */
+  public async associateClientWithSession(clientId: string): Promise<AccessTokenData> {
+    const storedToken = this.accessTokenStore.accessToken;
+    const newToken = await this.postAccess(storedToken, clientId);
+    return this.accessTokenStore.updateToken(newToken);
```

Comment:
The function does return new access token after associating it with current client, I wanted to make it more generic as it may be used in the future, wdyt?